Vagrantfile

```diff
@@ -273,4 +232,50 @@ Vagrant.configure(2) do |config|
             end
         end
     end
+    if ENV["NFS"] then
+        config.vm.synced_folder '.', '/home/vagrant/go/src/github.com/cilium/cilium', type: "nfs", nfs_udp: false
+        # Don't forget to enable this ports on your host before starting the VM
+        # in order to have nfs working
+        # iptables -I INPUT -p tcp -s 192.168.34.0/24 --dport 111 -j ACCEPT
```

Comment:
Did you check that in the case of TCP the ports match the UDP ones?

Reply:
@brb it's the same ports